### ISSUE TITLE
[Bug] Login Issues vpn

### ISSUE THREAD

**Issue by Princ229** [AppFlowy-IO/AppFlowy#5988]
### Bug Description

For about 24 hours now, I've been having trouble logging into my AppFlowy account. I've tried logging in via email and Google, but both methods are problematic:
When trying to log in via email, I never receive the login email
When trying to log in with Google, I get the error "Something went wrong. Please Try later"
I'm wondering if my account has been blocked.

### How to Reproduce

Attempt to log in via email
Attempt to log in with Google

### Expected Behavior

Able to log into my AppFlowy account without any errors.

### Operating System

Windows

### AppFlowy Version(s)

AppFlowy-0.6.7.1

### Screenshots

![Capture d’écran 2024-08-16 114738](https://github.com/user-attachments/assets/b438d3a8-7af6-4070-a50c-bf443a7a772e)

### Additional Context

If anyone has suggestions on how to resolve this login issue, I would greatly appreciate your help! I'm unable to access my account and am unsure why I'm encountering these errors when trying to log in. Please let me know if you have any ideas or if I should reach out to the AppFlowy support team. Any guidance would be very helpful to get me back into my account. Thank you in advance for your assistance!

**Comment by LucasXu0** [comment]
Hi, @Princ229. Can you share your log files with us? Here's the [guide](https://appflowy.com/a1b45051-39f5-469b-88da-d008ef40bc5a/AppFlowy-Q-A-efd81847-90da-4ca7-af28-9e7491c2d9ea) on how to export them.

**Comment by annieappflowy** [comment]
Please make sure the server is set to AppFlowy Cloud Beta
<img width="564" alt="image" src="https://github.com/user-attachments/assets/dde7f2bb-e8d5-402a-b10c-d22fc6089706">

**Comment by Princ229** [comment]
The log files

**Comment by Princ229** [comment]
I still don't have access to my account!
There is no way to recover my data from my work on this account? Otherwise, I'm in deep trouble without this data.

[log.zip](https://github.com/user-attachments/files/16642943/log.zip)

**Comment by Princ229** [comment]
> Assurez-vous que le serveur est configuré sur AppFlowy Cloud Beta <img alt="image" width="564" src="https://private-user-images.githubusercontent.com/12026239/358577130-dde7f2bb-e8d5-402a-b10c-d22fc6089706.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM4NTc1ODEsIm5iZiI6MTcyMzg1NzI4MSwicGF0aCI6Ii8xMjAyNjIzOS8zNTg1NzcxMzAtZGRlN2YyYmItZThkNS00MDJhLWIxMGMtZDIyZmM2MDg5NzA2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODE3VDAxMTQ0MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJmYzVjMWJlZDk4OTNlMWQ5ZjExODRlYjM2MGIyZDk4OGVlNzQ3OGU1YjBlNWI5MGQxMWU2YmJiMzEyZWI5ZmQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.xOQvlWAOYSnY5HQ49lqTqd4VgFzleuIgbv6zBm7HBzA">

I made the right choice

**Comment by annieappflowy** [comment]
@Princ229 , there are local copies on your device, so I wouldn't worry too much about them missing or lost.
We're checking your log files and will get back to you soon.

**Comment by khorshuheng** [comment]
Based on the logs, it seems that your AppFlowy application is not able to reach https://beta.appflowy.cloud/gotrue/token, https://beta.appflowy.cloud/gotrue/settings. By any chance, are you connecting to the internet behind a network proxy / VPN? Are you able to reach https://beta.appflowy.cloud/gotrue/settings from your browser?

**Comment by oxblixxx** [comment]
I am connected to vpn as the app is deployed internally, could this be an issue with authenticating after receiving the magic link in my email?

**Comment by khorshuheng** [comment]
We are using gotrue as our authentication service, which is reachable via  https://beta.appflowy.cloud/gotrue . Hence, if you are not able to reach this service, say, if this is blocked by a firewall, or the environment which the app is deployed on can't reach the internet for some reason, the sync and sign in won't work.

May i know what do you mean when you say that the app is deployed internally? Do you mean, you are deploying a self hosted appflowy cloud webservice? Or, the appflowy desktop application is installed on a remote desktop, and you have to access the appflowy desktop application via a VPN?

**Comment by oxblixxx** [comment]
I mean deploying a appflowy cloud webservice which is deployed behind a proxy

**Comment by khorshuheng** [comment]
Noted. If you are self-hosting appflowy cloud webservice, then on the cloud settings, you should choose `AppFlowy Self Hosted`, and specified the endpoint of the deployed appflowy cloud webservice, instead of AppFlowy Cloud Beta. Based on the log, the desktop application/mobile app is connecting to our hosted AppFlowy Cloud instead of your internally hosted appflowy cloud.

Before that, since you mentioned that you have important data on your account - does that mean that you have successfully log in in the past and created some document?

**Comment by Princ229** [comment]
By the way, I was visiting a friend, and it was there that he recommended
this to me. Since he uses a VPN, I connected through it to download the app
and navigate within it. I spent almost a month using his VPN for the
connection. However, it's been three days since I left his place, and I am
now using my own connection without a VPN. He doesn’t live far from me. If
I understand correctly, I need to reconnect to the VPN before I can access
my account.

Le sam. 17 août 2024 à 07:46, Khor Shu Heng ***@***.***> a
écrit :

**Comment by Princ229** [comment]
> D’après les logs, il semble que votre application AppFlowy ne soit pas en mesure [d’atteindre https://beta.appflowy.cloud/gotrue/token](https://beta.appflowy.cloud/gotrue/token), https://beta.appflowy.cloud/gotrue/settings. Par hasard, vous vous connectez à Internet derrière un proxy réseau / VPN ? Pouvez-vous https://beta.appflowy.cloud/gotrue/settings joindre depuis votre navigateur ?

Yes, I was able to access it, but here is what I have: {
    "external": {
        "apple": false,
        "azure": false,
        "bitbucket": false,
        "discord": true,
        "facebook": false,
        "figma": false,
        "fly": false,
        "github": true,
        "gitlab": false,
        "google": true,
        "keycloak": false,
        "kakao": false,
        "linkedin": false,
        "linkedin_oidc": false,
        "notion": false,
        "spotify": false,
        "slack": false,
        "workos": false,
        "twitch": false,
        "twitter": false,
        "email": true,
        "phone": false,
        "zoom": false
    },
    "disable_signup": false,
    "mailer_autoconfirm": false,
    "phone_autoconfirm": false,
    "sms_provider": "",
    "mfa_enabled": false,
    "saml_enabled": false
}

**Comment by Princ229** [comment]
Heyyyy

**Comment by khorshuheng** [comment]
Under normal circumstances, you shouldn't really need a VPN to access Appflowy cloud, unless you are using a self hosted instance that cannot be accessed without one, which is not the case here.

Let's try if signing in via the web portal works, since you were able to access the gotrue settings via browser. Go to https://beta.appflowy.cloud, which should then redirect you to a log in page, and let us know the result.

I have not been able to figure out why our API endpoint is reachable via your browser, but somehow not via your desktop application. One reason I can think of, is that the desktop application is somehow installed in a virtual environment that does not have access to the internet, though I doubt that is the case here.

**Comment by Princ229** [comment]
I was able to access the website, and here is what I have,
![Capture d’écran 2024-08-19 090523](https://github.com/user-attachments/assets/7431ef70-cda4-4b51-9240-9f4ef82e4c88)
 I clicked on the Open AppFlowy button, and it opened the application on my desktop, but I couldn't connect to my account; it’s the same error as before.

**Comment by Princ229** [comment]
Assistance is slow enough, I think. I need to log in to finish my work.
It's the only thing that's slowing him down now.

Le lun. 19 août 2024 à 01:55, Khor Shu Heng ***@***.***> a
écrit :

**Comment by khorshuheng** [comment]
@Princ229 I believe you accidentally closed this yesterday, let me reopen this issue.

What we know for sure at the moment, is that there isn't any issue with your account - but somehow your desktop app isn't able to reach our service.

Can you try and see if you are able to login using our android/ios mobile app?

**Comment by Princ229** [comment]
I tried the mobile application a few days ago but I heard the application
doesn't open it automatically closes. Maybe it's not compatible with my
Infinix.

**Comment by LucasXu0** [comment]
@Princ229 Could you let me know which VPN client you're using and check if your VPN client is set to use a proxy for the AppFlowy app? If so, try disabling the proxy, reopening the app, and trying to log in again.

**Comment by Princ229** [comment]
It's a VPN that he configured on my PC I don't know much about. But as I
understand it, do I reconnect it with ke VpN I can access the app?

**Comment by LucasXu0** [comment]
You can try reconnecting your VPN and logging in again. If it works, it means the application was under a network proxy, which was causing continuous connection failures.

**Comment by Princ229** [comment]
Ok, I will try.

**Comment by Princ229** [comment]
I was able to access my account when I used the VPN to reconnect, everything worked until the moment I realized that I don't have all my current work, the recent work I did is not present in any of the workspaces! The part I circled in the photo should have up to 07 other new spaces I created with very important projects! Can you explain to me why?? The strangest thing is that the images associated with these last spaces I created are indeed present in the folder, about fifteen images that I put in these spaces: AppData\Roaming\io.appflowy\AppFlowy\data_aHR0cHM6Ly9iZXRhLmFwcGZsb3d5LmNsb3Vk\cache_files but I don't see the corresponding spaces in my workspace
![Capture d’écran 2024-08-21 011924](https://github.com/user-attachments/assets/fcbfa89f-a14d-4818-8156-926249ed8307)
![Capture d’écran 2024-08-21 011816](https://github.com/user-attachments/assets/6e0df9e9-bc38-472e-9ce6-6682a20a5b13)

**Comment by khorshuheng** [comment]
Can you try using the search functionality to look for your recent documents?

**Comment by annieappflowy** [comment]
@LucasXu0

**Comment by Princ229** [comment]
I'm sorry, but it's not working. I've tried everything. I must first find
all the other 07 "Space" before I can find the said documents.

Le mer. 21 août 2024 à 02:25, Khor Shu Heng ***@***.***> a
écrit :

**Comment by LucasXu0** [comment]
> The part I circled in the photo should have up to 07 other new spaces I created with very important projects!

Do you remember when you created these 7 spaces? From your log files, you created 2 spaces respectively on 08-11 and 08-14.

<img width="284" alt="Screenshot 2024-08-21 at 10 22 45" src="https://github.com/user-attachments/assets/f353c7b5-549c-4099-aba3-813e177f8e86">

If you did that after 08-16, please share your latest log files and don't delete any local folders.

**Comment by Princ229** [comment]
J'ai bien plus que 02 Space je vous assure. Je me rappelle plus du nombre
mais ça dépasse 2.

**Comment by khorshuheng** [comment]
Were those space created before 11th of August? And just to confirm, have you login only using one email, or are there different emails that you used to login to AppFlowy?

**Comment by Princ229** [comment]
No, I think the majority was created after August 11th. I remember better
since it's supposed to be my second brain, isn't it?

Different email? Or different Computer? Otherwise it's true that the day I
lost access to my account I tried to log on to another PC but in vain

Le jeu. 22 août 2024 à 02:18, Khor Shu Heng ***@***.***> a
écrit :

**Comment by LucasXu0** [comment]
There are some things that should be clarified.

Computer A (used to send log files):

1. Created 2 spaces
2. Created multiple pages within these 2 spaces

**Note: Most operations were not synced with the server due to network connectivity issues.**

Computer B (shown in the last screenshot):

1. Created 2 different spaces
2. Created multiple pages within these 2 spaces

Important: **The spaces and pages on Computer B are distinct from those on Computer A**. It appears you created these spaces and pages on separate computers, unless you cleared all local data in your Computer A before logging in again.

To recover your data:

- Use the computer from which you sent the log files (Computer A)
- Connect to the VPN
- Log in to your account

This process should allow you to access the unsynced data stored locally on Computer A, unless you have cleared all the local data.

.

**Comment by Princ229** [comment]
Thanks, I'll check them out.

**Comment by gamefreshman** [comment]
Hello, I also have a similar problem, I also can not open the desktop software, but the mobile phone can be used normally. @khorshuheng

**Comment by gamefreshman** [comment]
I want to continue to know how to solve the problem

**Comment by khorshuheng** [comment]
> Hello, I also have a similar problem, I also can not open the desktop software, but the mobile phone can be used normally. @khorshuheng

Could you create another issue, and provide additional information over there, such as the type of OS, version of AppFlowy App, and whether this is a self hosting appflowy cloud instance?
